releasenotes/notes/num-qubits-streamlining-01a9f58684ca4609.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    The ``n_qubits`` and ``numberofqubits`` keywords are deprecated throughout 
+    Terra and replaced by ``num_qubits``.
```

Comment:
Honestly, I know it's a real pain but for this kind of thing it's good to list out exactly which user facing api has the deprecated arg and what they should migrate it to.

Reply:
I added a more detailed table :+1: